tools/scripts/docs-website/create-docs-pr.js

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+// const { program } = require('commander')
+
+const Github = require('./github')
+const git = require('./git-commands')
+
+const DEFAULT_FILE_NAME = 'CHANGELOG.md'
+/** e.g. v7.2.1 */
+// const TAG_VALID_REGEX = /v\d+\.\d+\.\d+/
+const TAG_VALID_REGEX = /v\d+/
+const BASE_BRANCH = 'develop'
+
+var options = require('yargs')
+    .usage('$0 [options]')
+
+    .string('t')
+    .alias('t', 'tag')
+    .describe('t', 'tag name to get version of released agent')
+
+    .string('r')
+    .alias('r', 'remote')
+    .describe('r', 'remote to push branch to')
+    .default('r', 'origin')
+
+    .string('u')
+    .alias('u', 'username')
+    .describe('u', 'Owner of the fork of docs-website')
+    .default('u', 'metal-messiah')
```

Comment:
Its not immediately obvious if there is an alternative, since the user and PAT that GH actions can generate are scoped only to the repo they were created in (and we need to be able to push to an external repo)